File: year2020/month10/day1001/ms_1001.cs
using System;

namespace algorithm_69
{
    class ms_1001
    {
        static void recursionPrint(int n) 
        {
            if(n == 0)
                return;
            
            Console.WriteLine("recursive");
            recursionPrint(n-1);
        }

        static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            if(0 < n && n <= 20) 
                recursionPrint(n);
        }
    }
}

File: year2020/month10/day1001/ms_1001.cpp
#include <iostream>

using namespace std;

void recursionPrint(int n);

int main() {
	int n;

	cin >> n;

	if (0 < n && n <= 20)
		recursionPrint(n);

	return 0;
}

void recursionPrint(int n) {
	if (n == 0)
		return;

	cout << "recursive" << endl;
	recursionPrint(n - 1);
}